Makefile

```diff
@@ -59,6 +59,10 @@ endif
 # use customized config file
 include $(config)
 
+ifeq ($(USE_MKL2017), 1)
+	USE_MKLDNN=1
```

Comment:
Print a message that we're automatically switching to MKLDNN and that the usage of this option is deprecated